src/main/scala/com/workday/warp/junit/TestIdSupport.scala

```diff
@@ -0,0 +1,36 @@
+package com.workday.warp.junit
+
+import org.pmw.tinylog.Logger
+
+/** Utility methods for constructing test ids.
+  *
+  * Created by tomas.mccandless on 6/17/20.
+  */
+trait TestIdSupport {
+  import TestIdSupport._
+
+  /**
+    * Parses a Junit unique id to obtain a test id.
+    *
+    * Example unique id:
+    *   [engine:junit-jupiter]/[class:com.workday.warp.junit.MeasurementCallbacksSpec]/[method:foo()]
+    *
+    * @param uid
```

Comment:
could you move the example unique id under the @param and fill out the return documentation?

Reply:
will do